pkg/cluster/ignition/library.go

```diff
@@ -252,11 +253,18 @@ func clusterCommon(cc *clusterCommonConfig, action string) (*igntypes.Config, er
 	}
 
 	// Update service configuration file
+	ostreeTransport, registry, tag, err := image.ParseOstreeReference(cc.OsRegistry)
+	if err != nil {
+		return nil, err
+	}
+	if tag != "" {
+		return nil, fmt.Errorf("osRegistry field cannot have a tag")
+	}
 	updateFile := &File{
 		Path: OcneUpdateConfigPath,
 		Mode: 0400,
 		Contents: FileContents{
-			Source: fmt.Sprintf(OcneUpdateYamlPattern, cc.OsRegistry, cc.OsTag),
```

Comment:
There needs to be handling for cases where an ostree transport is not provided in osRegistry, matching the polite behavior in `ostree image create`